administration/src/cards/extensions/EMailNotificationExtension.ts

```diff
@@ -1,24 +1,19 @@
 import { Extension } from './extensions'
 
-class EMailNotificationExtension extends Extension<string, null> {
-  public readonly name = EMailNotificationExtension.name
+export const EMAIL_NOTIFICATION_EXTENSION_NAME = 'emailNotification'
+type EmailNotificationExtensionState = { [EMAIL_NOTIFICATION_EXTENSION_NAME]: string }
 
-  setInitialState(): void {}
-  createForm(): null {
-    return null
-  }
-  causesInfiniteLifetime(): boolean {
-    return false
-  }
-  fromString(state: string): void {
-    this.state = state
-  }
-  toString(): string {
-    return this.state ?? ''
-  }
-  isValid(): boolean {
-    return true
-  }
+const EMailNotificationExtension: Extension<EmailNotificationExtensionState> = {
+  name: EMAIL_NOTIFICATION_EXTENSION_NAME,
+  Component: () => null,
+  // TODO Why initial null?
+  getInitialState: () => ({ [EMAIL_NOTIFICATION_EXTENSION_NAME]: '' }),
+  causesInfiniteLifetime: () => false,
+  // TODO Why no protobuf data?
```

Comment:
Why is this initially set to null?

Reply:
if it will not be passed as a queryParam or from the applications view it will be null since we have no information about email in some cases when cards will be created directly without application (nuernberg project)